src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -87,3 +119,303 @@ export const validateStorageSubmissionParams = celebrate({
     version: Joi.number().required(),
   }),
 })
+
+export const createFormsgReqBody: SharedSubmissionMiddlewareHandlerType =
+  async (req, res, next) => {
+    if (req.body.formsg) return res.send(new FormsgReqBodyExistsError())
```

Comment:
qn: is this to guard against the case where middleware is called again later and potentially overrides an existing req.body.formsg?
- if yes, may want to use typing to guard against, instead of returning error at runtime

Reply:
Its more of guarding against future middlewares that might be added and happen to use the same namespace. We want to be able to detect and guard against that so that we don't have unintended consequences in the future.